src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx

```diff
@@ -0,0 +1,188 @@
+import { EntityHeader, Sans, Spacer } from "@artsy/palette"
+import { RecommendedArtist_artist } from "__generated__/RecommendedArtist_artist.graphql"
+import { track } from "Artsy/Analytics"
+import * as Schema from "Artsy/Analytics/Schema"
+import { SystemContext } from "Artsy/SystemContext"
+import { FillwidthItem } from "Components/Artwork/FillwidthItem"
```

Comment:
Oh yea, since you're doing the `...FillwidthItem_artwork` thing below, this has to be a fragment container, but you're importing the React component. If you change this to `import FillwidthItem` (it's the [default export](https://github.com/artsy/reaction/blob/686719aa1c8594a8a6baa73312e7c13c77b2c7ca/src/Components/Artwork/FillwidthItem.tsx#L133) for some reason) , then you won't need the `@relay(mask: false)` anymore and it should all work.

Reply:
I cargo-culted the FillwidthItem stuff from the RecentlyViewed carousel - I'll investigate whether I need it or not!

Reply:
Thanks for the tip! This worked.